docs/integrations/sources/gmail.md

```diff
@@ -0,0 +1,38 @@
+# Gmail
+Gmail is the email service provided by Google.
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `client_id` | `string` | OAuth Client ID.  |  |
+| `client_secret` | `string` | OAuth Client Secret.  |  |
+| `client_refresh_token` | `string` | Refresh token.  |  |
+| `includeSpamTrash` | `boolean` | Include Spam &amp; Trash. Include drafts/messages from SPAM and TRASH in the results. Defaults to false. | false |
```

Comment:
@topefolorunso can you re-upload and snake_case_the_config_input_name here? I.e. `include_spam_and_trash`.

Reply:
Ah, you know what, I think I will search and replace it here instead. One sec.

Reply:
i'll need to change that in manifest too. one sec